splunk/common-files/createdefaults.py

```diff
@@ -43,8 +45,13 @@ def random_generator(size=24):
     os.environ["SPLUNK_PASSWORD"] = random_generator()
 if not splunk_idxc_secret:
     os.environ["SPLUNK_IDXC_SECRET"] = random_generator()
+if not splunk_idxc_pass4SymmKey:
+    os.environ["SPLUNK_IDXC_SECRET"] = os.environ["SPLUNK_IDXC_SECRET"]
```

Comment:
Thinking about this more, maybe this section should first check `splunk_idxc_pass4SymmKey`, that way when do deprecate this variable in the splunk-ansible code, we don't need to modify this script again. Something like
```
if splunk_idxc_pass4SymmKey:
    os.environ["SPLUNK_IDXC_PASS4SYMMKEY"] = os.environ["SPLUNK_IDXC_SECRET"] = splunk_idxc_pass4SymmKey
elif splunk_idxc_secret:
    os.environ["SPLUNK_IDXC_PASS4SYMMKEY"] = os.environ["SPLUNK_IDXC_SECRET"] = splunk_idxc_secret
else:
   # generate random

Reply:
Makes sense